content/docs/user-guide/packaging/content-patch-package.md

```diff
@@ -0,0 +1,220 @@
+---
+linkTitle: Content Patch Package
+title: Creating a Content Patch Update Package
+description: Learn how to create an Open 3D Engine (O3DE) content update patch package for your released project.
+toc: true
+weight: 300
+---
+
+This topic explains the process for creating a content patch update package for your **Open 3D Engine (O3DE)** project with **Asset Bundler**. This includes a patch bundle that contains just the asset updates, and a patched release bundle that contains all the game assets including the asset updates.
+
+## Prerequisites
+
+To follow along, you'll need a project with a release build that has been created through the steps in the previous topic, [Creating a Project Game Release Layout for Windows](windows-release-builds).
+
+## Conventions used in this topic
+
+This topic makes some assumptions about your project for clarity:
+
+* Your project has a seed list file and an asset list for game only assets.
+* You are using a source engine build within your project.
+
+The following conventions are used in this topic as substitutions for your own directory and file names:
+
+* Project name and directory: `C:\MyProject`
+* Source engine build directory: `C:\MyProject\build\windows\bin\profile`
+* Game asset seed list : `game_seed_list.seed`
+* Game asset list: `game_pc.assetlist`
+
+## Make asset changes to your project
+
+Make a visual change to your project that simulates a content update. For demonstration purposes, you might add an asset to a level that is easy to spot when you test the update bundle. Save the changes to your level.
+
+## Create an update package
+
+There are four steps to creating an update package:
+
+* Version the old asset lists.
+* Generate new asset lists based on the changes.
+* Create a patched release asset bundle that contains all the game assets and updates.
+* Create a patch asset bundle that contains just the game asset updates.
+
+{{< note >}}
+For most updates, only the game assets will change. When upgrading to a new version of O3DE and pushing an update, you should regenerate all of the existing content to ensure it's properly updated, including the engine assets and auxiliary content.
+{{< /note >}}
+
+### Version the asset list
+
+You must version the existing asset list so that it can be compared to the updated asset list. You can do this by simply adding a version identifier to the name of the existing asset list.
+
+1. Add a version identifier to the existing `game_pc.assetlist` in your project's `AssetBundling\AssetLists\` directory:
```

Comment:
I would rather these steps be removed from this tutorial, and instead the first tutorial on asset bundling be updated to include a version in the filename of the asset list.